examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Suggestions:</p><ul><li>Simplify the title to "Topological feature extraction from graphs"</li><li>Change "github" to "GitHub"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Agree with both!</p>

Reply:
<p>Done now.</p>